Prepare information for release 0.1.0. (#67)
ios-course/ironfoudation-team-project: release_notes.md

The file describes all the changes, that are affected the usage of the application.

The format is based on [Release_notes format](https://github.com/ios-course/ironfoudation-team-project/wiki/Release_notes-format).

> ## [0.1.0] - [29.05.2023]

Reviewer: What is the update date means? :)

> ## [0.1.0] - [29.05.2023]

Author: I think it is synonymous.
As we discussed with team for avoiding duplicated info the release date in this file are deleted.

### New features

> `Quicklook`: Users can check a list of pet sitters available for pet owners. Each list row includes an avatar, full name, price, short bio, and pet sitter rating.

Reviewer: ```suggestion
`Quicklook`: Pet owners can see a list of available pet sitters. <IS THE LIST STATIC/DYNAMIC?> For every pet sitter the following information is available: an avatar icon, full name, current rate, short bio, and rating.
```

Author: You can scroll the list up and down. So it's dynamic list.
Thank you for suggestion. Fixed it.

Author: Thanks for the explanation! In this release it is fixed list.

</details>

<details>

> <summary> CLICK HERE</summary>

Reviewer: nit:
```suggestion
<summary>See the screenshot of the UI</summary>
```

<p>

> ![list of pet sitters](https://github.com/ios-course/ironfoudation-team-project/assets/120586844/a62ea90f-50f1-4c44-81c1-e0ef8091feae)

Reviewer 2: nit:
it is better to scale the screenshots so that they do not occupy the entire screen space in height
```suggestion
 <img src="https://github.com/ios-course/ironfoudation-team-project/assets/120586844/a62ea90f-50f1-4c44-81c1-e0ef8091feae" width=50% height=50%>
```

Author: Thanks for useful comment!
I've deleted this screenshot, according to suggestion from Dersim.

</p>
</details>

> `Get detailed information about the pet sitter`: The app allows users to get more information on the tap of the pet sitter's row in the list like the pet sitter's phone number, rating, and full bio.

Reviewer: ```suggestion
`Get detailed information about the pet sitter`: The app allows pet owners to get more information about the pet sitter by tapping them in the list.```

Author: Thanks, fixed it.

</details>

<details>
<summary> CLICK HERE</summary>

<p>

![sitter](https://github.com/ios-course/ironfoudation-team-project/assets/120586844/4994d3ef-8b12-4083-8924-1730c9b53ef6)

</p>
</details>

> ### Notes

Reviewer: Why do we need this info here?

Author: Yes, this info information doesn't apply to this file. I deleted it.

The file describes all the changes, that are affected the usage of the application.

The format is based on [Release_notes format](https://github.com/ios-course/ironfoudation-team-project/wiki/Release_notes-format).

[0.1.0] - [26.05.2023]

> 

Author: Fixed it.

### New features

> `Quicklook`:  Users can check a list of pet sitters available for pet owners. Each row of the list includes an avatar, name + surname, price, short bio and rating of pet sitters.

Author: Thanks, fixed it.

</details>

<details>
<summary> CLICK HERE</summary>

<p>

![list of pet sitters](https://github.com/ios-course/ironfoudation-team-project/assets/120586844/a62ea90f-50f1-4c44-81c1-e0ef8091feae)

</p>
</details>

> `Remote check the pet sitter`: App allows users without internet connection tab to the pet sitters' row in the list of sitters and read more detailed information about the sitter:  pet sitter's phone number, rating and full bio.

Reviewer 3: For me, it's a confusing phrase `Remote check the pet sitter`. I can check what the pet sitter does somehow 😕 
In the future user can't get the list of pet sitters or check detailed information about it without an internet connection.
The main idea of the detail screen is to show more information than it provides in the card view in the list, that's it.
```suggestion
`Get detailed information about the pet sitter`: The app allows users to get more information on the tap of the pet sitter's row in the list like the pet sitter's phone number, rating, and full bio.
```

Author: Agree, thank u. Fixed it.

</details>

<details>
<summary> CLICK HERE</summary>

<p>

![sitter](https://github.com/ios-course/ironfoudation-team-project/assets/120586844/4994d3ef-8b12-4083-8924-1730c9b53ef6)

</p>
</details>

> `Easy to navigate`:  Users can easily switch between pet sitters' list and the pet sitter window with detailed information.

Reviewer 3: Does it make sense to mention this?
It's standard navigation.

Author: Yes, sure. The line are deleted.

> ### Notes
> - [General info, installation, authors](https://github.com/ios-course/ironfoudation-team-project/blob/main/README.md)

Reviewer 3: Why do we need a list with one bullet point?
```suggestion
### Notes
[General info, installation, authors](https://github.com/ios-course/ironfoudation-team-project/blob/main/README.md)
```

Author: Fixed it.
